### DISCUSSION TITLE
Notification Dismiss Android

### DISCUSSION THREAD

**Discussion by juzejunior** [jhomlala/betterplayer#632] [Q&A]
Hello guys, thanks for the great work!
I`ve been using version 0.64, and when I close the while i`m playing some video the notification bar remains and cannot be dismissed. I have implemented the service described to kill notifications, but it doesn`t work.  Has anyone gone through this?

**Comment by jhomlala** [comment]
Notification can't be removed until your app is running. If you've any problem, please report it in issues.

**Comment by juzejunior** [comment]
Understand. My question would be if this is a default behavior, in case the notification remains active when I kill the app while watching a video, or if it would be a bug.
